Packs/CommonWidgets/Scripts/GetLargestInvestigations/GetLargestInvestigations.py

```diff
@@ -0,0 +1,83 @@
+import traceback
+from typing import List, Dict
+from operator import itemgetter
+from dateutil.parser import parse
+from dateutil.relativedelta import relativedelta
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+def get_investigations(raw_output, investigations):
+    for db in raw_output:
+        buckets = db.get('buckets')
+        for entry in buckets.keys():
+            if entry.startswith('investigations-'):
+                investigations[entry] = buckets.get(entry)
+                investigations[entry].update({"Date": db.get('dbName')})
+
+
+def parse_investigations_to_table(investigations):
+    data: List = []
+    widget_table = {"total": len(investigations)}
+    for investigation in investigations.keys():
+        size = investigations[investigation].get('leafSize').split(' ')
+        if float(size[0]) >= 1.0 and size[1] == 'MB':
+            db_name = investigations[investigation].get('Date')
+            data.append({
+                "IncidentID": investigation.split('-')[1],
+                "Size": investigations[investigation].get('leafSize'),
+                "AmountOfEntries": investigations[investigation].get('keyN'),
+                "Date": db_name[:2] + "-" + db_name[2:]
+            })
+
+    widget_table['data'] = sorted(data, key=itemgetter('Size'), reverse=True)  # type: ignore
+
+    return widget_table
+
+
+def get_month_db_from_date(date):
+    month = date.strftime('%m')
+    year = date.strftime('%Y')
```

Comment:
why only in months?

Reply:
you can only filter the `getDBStatistics` command by db names which are effectively months (for example 082020). There are no further date-time info about the incidents from the output so this is the best I can do for time control.